Signal/src/AppDelegate.m

```diff
@@ -306,7 +306,8 @@ - (void)verifyDBKeysAvailableBeforeBackgroundLaunch
         [[UIApplication sharedApplication] scheduleLocalNotification:notification];
         [UIApplication.sharedApplication setApplicationIconBadgeNumber:1];
 
-        OWSFail(@"!isDatabasePasswordAccessible.");
+        [DDLog flushLog];
```

Comment:
Shouldn't we log something before we exit?

Reply:
Existing logging is up a couple lines, we just want to make sure it (and anything else) flushes before exiting.